ocis-pkg/log/log.go

```diff
@@ -44,6 +51,32 @@ func setMicroLogger() {
 	)
 }
 
+// setDefaultLogger creates an environment-specific logger and sets it as the default logger.
```

Comment:
as discussed I do not really like that here. We should do this with #8259.

Reply:
i need an logger (i do not want to rely on fmt.PrintX here, because of the log level), lets do it that way, we keep it here... and we do a sequence pr in which we rebuild all the places that could use a default logger... I assume we can clean up a lot of places that have a logger passed as a function parameter.

another solution could be, do not have a default logger and use fmt or slog instead, a lot of places like ocis-pkg are loose and have no access to a dedicated logger.

thoughts?